src/sentry/static/sentry/app/views/eventsV2/table/cellAction.tsx

```diff
@@ -120,7 +120,11 @@ class CellAction extends React.Component<Props, State> {
       }
     }
 
-    if (column.type !== 'duration') {
+    if (
+      column.type !== 'duration' &&
+      column.type !== 'number' &&
+      column.type !== 'percentage'
```

Comment:
Should we include integer type columns here too?

Reply:
Not 100% sure but it feels like being able to filter for a specific integer might be a useful thing unlike floats.